app_dart/lib/src/request_handlers/get_branches.dart

```diff
@@ -6,9 +6,11 @@ import 'dart:async';
 
 import 'package:cocoon_service/src/model/appengine/branch.dart';
 import 'package:process_runner/process_runner.dart';
+import 'package:github/github.dart' hide Branch;
```

Comment:
Sure, changed to prefix. 

Umm for my learning purposes, how would dependabot rolls affect imports? thank you!

Reply:
If one of the packages was to add a new class that's already in the package:github namespace, it'll cause collisions and block the autoroller. Since github is a large library, it's a good candidate for prefixing.

Reply:
Thanks for explaining!